### PR TITLE
Remove SSREntity table from WCAndroidDatabase, as well as its related usages.

### PR DESCRIPTION
**Pull request by hafizrahman**
The System Status Report feature in WooCommerce Android ended up not needing to use any persistence, since we always want the SSR info to be fresh for troubleshooting purposes.

The database content has never been used in WooCommerce Android in the first place, and this PR removes the table and its usages.

### CODE REVIEW

File: plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/persistence/WCAndroidDatabase.kt [hunk 1]
```diff
@@ -47,7 +43,7 @@ abstract class WCAndroidDatabase : RoomDatabase() {
                 "wc-android-database"
         ).allowMainThreadQueries()
                 .fallbackToDestructiveMigrationOnDowngrade()
-                .addMigrations(MIGRATION_3_4)
+                .addMigrations(MIGRATION_4_5)
```
Comment (wzieba) [inline plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/persistence/WCAndroidDatabase.kt:48]: We have to add all migrations, not change the current one. Currently, if someone has app with version 3 and then installs with version 5, the app will crash.

**Review by wzieba — CHANGES_REQUESTED** [review]
Thanks @hafizrahman for keeping DB clean! I've added one comment but crucial one. Other than that, all looks and works good.

File: plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/persistence/WCAndroidDatabase.kt (hunk 1 above)
Comment (hafizrahman) [reply plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/persistence/WCAndroidDatabase.kt:48]: > We have to add all migrations

Aha! I misunderstood this before, thank you!

**Review by wzieba — APPROVED** [review]
Thanks @hafizrahman !

**Pull request merged**